packages/eslint-config-typescript/README.md

```diff
@@ -0,0 +1,43 @@
+# @celo/eslint-config-typescript
+
+Shareable ESLint config for Celo TypeScript packages.
+
+## Installing
+
+Add the ESLint config in your package:
+
+```
+yarn add @celo/eslint-config-typescript --dev
+```
+
+and add all the [`peerDependencies`](./packages.json).
```

Comment:
This is a requirement due to how eslint does [plugin resolution](https://eslint.org/docs/user-guide/configuring/plugins#configuring-plugins):

> Plugins are resolved relative to the config file. In other words, ESLint will load the plugin as a user would obtain by running require('eslint-plugin-pluginname') in the config file.

> Plugins in the base configuration (loaded by extends setting) are relative to the derived config file. For example, if ./.eslintrc has extends: ["foo"] and the eslint-config-foo has plugins: ["bar"], ESLint finds the eslint-plugin-bar from ./node_modules/ (rather than ./node_modules/eslint-config-foo/node_modules/) or ancestor directories. Thus every plugin in the config file and base configurations is resolved uniquely.